app/test/integration-test/src/test/java/io/syndesis/test/itest/googlecalendar/WebhookToGoogleCalendar_IT.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.test.itest.googlecalendar;
+
+import io.syndesis.test.SyndesisTestEnvironment;
+import io.syndesis.test.container.integration.SyndesisIntegrationRuntimeContainer;
+import io.syndesis.test.itest.SyndesisIntegrationTestSupport;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import com.consol.citrus.annotations.CitrusResource;
+import com.consol.citrus.annotations.CitrusTest;
+import com.consol.citrus.dsl.runner.TestRunner;
+
+public class WebhookToGoogleCalendar_IT extends SyndesisIntegrationTestSupport {
+
+    /**
+     * Webhook receives POST request and creates a Google Calendar event.
```

Comment:
Cool new integration test! We could also receive the created Calendar event in the test and verify its content

Reply:
We can't really, look at the commit messages.

Reply:
What is the output of the running integration then? Where is the calendar event stored? Isn't that sent to the mocked `www.googleapis.com`?

Reply:
I can't stub the `www.googleapis.com`, the resolution I can influence but the host port forwarded ends up being in the `sshd` container spun up by testcontainers and I don't know the IP of that container. Also the Jetty setup using TLS didn't seem to work.